scripts/nginx/mylar.sh

```diff
@@ -0,0 +1,430 @@
+#!/bin/bash
+# Nginx Configuration for Mylar
+# Author: Public920
+
+user=$(cut -d: -f1 < /root/.master.info)
+
+isactive=$(systemctl is-active mylar)
+if [[ $isactive == "active" ]]; then
+    systemctl stop mylar
+fi
+
+if [[ ! -f /etc/nginx/apps/mylar.conf ]]; then
+    cat > /etc/nginx/apps/mylar.conf <<MYNG
+location /mylar {
+    include /etc/nginx/snippets/proxy.conf;
+    proxy_pass http://127.0.0.1:8090/mylar;
+    auth_basic "What's the password?";
+    auth_basic_user_file /etc/htpasswd.d/htpasswd.${user};
+}
+MYNG
+fi
+cat > /opt/mylar/config.ini <<MYCONF
```

Comment:
Seeing as this is in the nginx folder, this will only get triggered if:
User has nginx installed before the app
User installs nginx after the app has been installed.

THerefore I'm not sure this is the place to cat out a config that's specifically for `mylar` and has not much to do with the nginx. If you don't have nginx, you won't get this config file as it is right now.